src/analysis/souffle/taint.dl

```diff
@@ -77,6 +78,16 @@
 // Rules
 //-----------------------------------------------------------------------------
 
+// Draw an edge from any operation's operand to its result.
+resolvedEdge(owner, src, tgt) :- operationToOperands(owner, _, tgt, src).
```

Comment:
Shouldn't these edges be added only for those operations for which we don't understand how edges propagate. e.g., if we have a special interpretation for  "*" and if we  add these edges, won't we propagate tags imprecisely?

Reply:
We won't. There are two cases here: `ConfidentialityTag`s and `IntegrityTag`s:

* For the `ConfidentialityTag` case, imprecise overpropagation anywhere where dataflow is plausible is what we want, so this is fine.

* For the `IntegrityTag` case, we draw an additional edge from the `ArbitraryComputation` `AccessPath` to the result of any `operation` other than `=`. `ArbitraryComputation` has no incoming edges and has no `IntegrityTag`s. Because `IntegrityTag`s only propagate to an `AccessPath` when all incoming edges start at `AccessPath`s that have that `IntegrityTag`, default propagation of `IntegrityTag`s is blocked across all non-`=` operations.